pkg/cosign/keys.go

```diff
@@ -73,7 +150,7 @@ func GenerateKeyPair(pf PassFunc) (*Keys, error) {
 	// store in PEM format
 	privBytes := pem.EncodeToMemory(&pem.Block{
 		Bytes: encBytes,
-		Type:  PrivakeKeyPemType,
+		Type:  PrivateKeyPemType,
```

Comment:
note for self: this is a sneaky fix for 'privake" to 'private'. What're the consequences of this fix in terms of data dependency?

Reply:
I believe this was a typo.